internal/cmd/search-integration-tester/search_tests.go

```diff
@@ -170,7 +174,7 @@ func runSearchTests() error {
 		filename := strings.ToLower(sanitizeFilename(test.Name))
 		goldenPath := path.Join(searchTestDataDir, fmt.Sprintf("%s.golden", filename))
 		if err := assertGolden(test.Name, goldenPath, got, update); err != nil {
-			return err
+			return fmt.Errorf("TEST FAILURE: %s\n%s", test.Name, err.Error())
```

Comment:
```suggestion
			return fmt.Errorf("TEST FAILURE: %s\n%s", test.Name, err)
```

It'll automatically convert error to string for you.

Reply:
Nice. This is going to be useful for something else :-)